recipes/ngsutils/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: ngsutils
+  version: '0.5.9'
+
+source:
+  url: https://github.com/ngsutils/ngsutils/archive/ngsutils-0.5.9.tar.gz
+  md5: 6212d6de740c42aa2c6b5f6785465860
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python >=2.7.0
+    - pysam
+    - cython
+    - samtools
+    - eta
+    - coverage
+    - swalign
+  run:
+    - python >=2.7.0
+    - pysam
+    - cython
+    - samtools
+    - eta
+    - coverage
+    - swalign
+
+test:
+  commands:
+    - python -m unittest discover ngsutils
+
+about:
+  home: http://ngsutils.org
+  license: LICENSE
```

Comment:
Actually these particular licenses require that the license files are distributed with the packages. I think a solution for this is to package the licenses with the recipe as suggested by conda-forge (http://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually).